src/instructlab/system/info.py

```diff
@@ -29,10 +29,13 @@ def _platform_info() -> typing.Dict[str, typing.Any]:
 
     if sys.platform == "linux":
         os_release = platform.freedesktop_os_release()
-        for key in ["ID", "VERSION_ID", "PRETTY_NAME"]:
+        for key in ["ID", "VERSION_ID", "PRETTY_NAME", "VARIANT"]:
             value = os_release.get(key)
             if value:
                 d[f"os-release.{key}"] = value
+        variant_id = os_release.get("VARIANT_ID")
+        if variant_id:
+            d["os-release.PRODUCT_NAME_VERSION_ID"] = variant_id
```

Comment:
I could be misreading, but this doesn't seem like what is being suggested in https://github.com/instructlab/instructlab/issues/2720

I thought it was suggesting you would be looking for:

`os_release.get(f"os-release.{variant_id.upper()}_VERSION_ID")`

and if it's set then set:

`d[f"os-release.{variant_id.upper()}_VERSION_ID"] = <to its value>`

test case could be useful here as well.

Reply:
sorry, just found it:
```
VARIANT="TEST TEST"
VARIANT_ID=test_test
TEST_TEST_VERSION_ID='1.1.1'
```